crates/pop-common/src/rust_writer_helpers/sample_files/runtime_using_construct_runtime_macro.rs

```diff
@@ -0,0 +1,258 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Same as above, to avoid introducing extra new code (even if it's just sample files for testing), you can use something like this: https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-parachains/src/new_parachain.rs#L102